src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -1,61 +1,69 @@
 import json
 import os
 import shutil
+import time
 import uuid
 from pathlib import Path
+from typing import List
 from warnings import WarningMessage
 
 import datajoint as dj
+import numpy as np
 import spikeinterface as si
 import spikeinterface.toolkit as st
 
 from ..common.common_interval import IntervalList
 from ..common.common_nwbfile import AnalysisNwbfile
 from ..common.dj_helper_fn import fetch_nwb
+from .merged_sorting_extractor import MergedSortingExtractor
 from .spikesorting_recording import SpikeSortingRecording
 from .spikesorting_sorting import SpikeSorting
 
 schema = dj.schema('spikesorting_curation')
 
 
+def apply_merge_groups_to_sorting(sorting: si.BaseSorting, merge_groups: List[List[int]]):
+    # return a new sorting where the units are merged according to merge_groups
+    # merge_groups is a list of lists of unit_ids.
+    # for example: merge_groups = [[1, 2], [5, 8, 4]]]
+
+    return MergedSortingExtractor(parent_sorting=sorting, merge_groups=merge_groups)
+
+
 @schema
 class Curation(dj.Manual):
```

Comment:
I would suggest naming this `CurationTask` to distinguish it from `AutomaticCuration`

Reply:
That name doesn't seem intuitive to me, as to me a task is something to be done. A curation is an arraignment of things, and that seems appropriate to me. But if others agree we can change this. 

We could also change the name of AutomaticCuration, but I'm fine with it as is, and it does automatically insert something into the Curation table.